src/msvc_helper_main-win32.cc

```diff
@@ -113,18 +113,18 @@ int MSVCHelperMain(int argc, char** argv) {
     PushPathIntoEnvironment(env);
   }
 
-  char* command = GetCommandLineA();
-  command = strstr(command, " -- ");
-  if (!command) {
+  wchar_t* command = GetCommandLineW();
+  command = wcsstr(command, L" -- ");
+  if(!command){
     Fatal("expected command line to end with \" -- command args\"");
   }
-  command += 4;
+  command +=4;
 
   CLWrapper cl;
   if (!env.empty())
     cl.SetEnvBlock((void*)env.data());
   string output;
-  int exit_code = cl.Run(command, &output);
+  int exit_code = cl.Run(WideToUtf8(command).c_str(), &output);
```

Comment:
Wont this end up getting converted back to Wide later?

I suppose that for the sake of not causing lots of changes to Ninja it makes sense to convert it, but it does seem wasteful from a high level perspective.

Reply:
This is for sake of changing as little as possible for now